src/ClientInterface.php

```diff
@@ -108,7 +108,7 @@
  * @method Status            ftaliasdel(string $alias)
  * @method Status            ftaliasupdate(string $alias, string $index)
  * @method Status            ftalter(string $index, Schema $schema, ?AlterArguments $arguments = null)
- * @method Status            ftcreate(string $index, Schema $schema, ?CreateArguments $arguments = null)
+ * @method Status            ftcreate(string $index, FieldInterface[] $schema, string $on = '', array $prefixes = [], string $filter = '', string $language = '', string $languageField = '', float $score = 0, string $scoreField = '', string $payloadField = '', bool $maxTextFields = false, int $temporary = 0, bool $noOffsets = false, bool $noHl = false, bool $noFields = false, bool $noFreqs = false, array $stopWords = [], bool $skipInitialScan = false)
```

Comment:
The [default](https://redis.io/commands/ft.create/) score should be 1.0 not 0

Reply:
It have 2 perspectives:

From Redis server perspective if `default_score` parameter isn't set it will have default value `1.0`. 
From client perspective, if you want to keep the default value you should pass nothing as argument in this case `0`, which means that this value won't be included in argument set so Redis server will use `1.0` as default.

Isn't it?

Reply:
No, I disagree. I think we should have the client, echo the default value of the server (i.e use 1.0). Otherwise, we end up with possible confusion. IMHO the experience is better this way  (aligned with Redis), especially when a user reads the command docs.

Reply:
Okay 👍

Reply:
@chayim Fixed